### PR TITLE
Rename configure.in to configure.ac

### PR DESCRIPTION
**Pull request by apteryks**

### CODE REVIEW

File: acinclude.m4
Comment (vadz) [inline acinclude.m4:387]: Ugh, sorry, but you really, really shouldn't be running automatic replacements in the entire search tree and if you do it, you should at least be using the right pattern, with `.` metacharacter being escaped.

**Review by vadz — CHANGES_REQUESTED** [review]
Sorry, this is another not ideal PR, it shouldn't include #23640 once again. Please either

1. Wait until the other PR is merged (or rejected).
2. Add this commit to the other PR itself.
3. Cherry pick just the new commit here and create a new PR with it (this will create conflicts when merging however).

Also, please carefully review your changes before submitting them, I'm sorry but I don't have enough time to do manual reviews of automatic replacements.

Thanks in advance.

File: acinclude.m4
Comment (Apteryks) [reply acinclude.m4]: Oops, sorry! I did quickly review the 54 something instances but missed that one.  I've fixed that typo, tightened the regexp and pushed, after validating that it seems to configure and build fine on my end.

**Pull request closed without merging**
